framework/db/schema/mssql/CMssqlCommandBuilder.php

```diff
@@ -204,7 +204,22 @@ public function applyLimit($sql, $limit, $offset)
 	}
 
 	/**
-	 * Rewrite sql to apply $limit > and $offset > 0 for MSSQL database.
+	 * @param string $sql sql query
+	 * @param integer $limit $limit > 0
+	 * @param integer $offset $offset > 0
+	 * @return string modified sql query applied with limit and offset.
+	 */
+	protected function rewriteLimitOffsetSql($sql, $limit, $offset)
+	{
+		if (version_compare($this->dbConnection->getServerVersion(), '11', '<')) {
```

Comment:
Yii 1 framework uses a bit older formatting standard, such as no space after if and no brackets for single line statements. Could you compare to other framework code and adjust accordingly?